resources/META-INF/plugin.xml

```diff
@@ -8,9 +8,9 @@
 
   <category>Custom Languages</category>
 
-  <version>18.4</version>
+  <version>19.0</version>
 
-  <idea-version since-build="171.1" until-build="173.*"/>
+  <idea-version since-build="172.1" until-build="173.*"/>
```

Comment:
Please keep this at 171.1. It causes me much grief if it is 172.1.

Reply:
Oddly, I didn't do this intentionally.  Is it possible that `ant plugin.xml` did?

Reply:
Probably. I think the build script is still producing two files but we're only publishing the zip file.